docs/Installing.asciidoc

```diff
@@ -254,6 +254,52 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Auditing - tracking openQA changes
+----------------------------------
+[id="auditing"]
+
+Auditing plugin enables openQA administrators to maintain overview about what is happening with the system.
+Plugin records what event was triggered by whom, when and what the request looked like. This includes
+tracking openQA workers which are tracked under user whose API keys is worker using.
+
+Audit log is directly accessible from +Admin menu+.
+
+Auditing, by default enabled, can be disabled by global configuration option in +/etc/openqa/openqa.ini+:
+[source, ini]
+--------------------------------------------------------------------------------
+[global]
+audit_enabled = 0
+--------------------------------------------------------------------------------
+
+Audit section of +/etc/openqa/openqa.ini+ allows to exclude some events from logging using
+space separated blacklist:
+[source, ini]
+--------------------------------------------------------------------------------
+[audit]
+blacklist = job_grab job_done
+--------------------------------------------------------------------------------
+
+List of events tracked by auditing plugin:
+----
+Assets:
+  asset_register asset_delete
+Workers:
+  worker_register command_enqueue
+Jobs:
+  iso_create iso_delete iso_cancel
+  jobtemplate_create jobtemplate_delete
+  job_create job_grab job_delete job_update_result job_done jobs_restart job_restart job_cancel job_duplicate
+  jobgroup_create jobgroup_connect 
+Tables:
+  table_create table_update table_delete
+Users:
+  user_comment user_login
+Needles:
+  needle_delete needle_modify
+----
+
+Some of these events are very common and may clutter audit database. For this reason +job_grab+ and +job_done+ events are blacklisted by default.
```

Comment:
Since I didn't implement anything like whitelist, the default blacklisting means that indeed they are included in distributed openqa.ini. You are right that's too late for installed instances, but there is no damage to be done. Some will even want to log even these events.